index.html

```diff
@@ -4,36 +4,115 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
     <title>TripHouse</title>
+    <link rel="icon" href="./images/favicons/favicon.ico" />
     <link rel="stylesheet" href="styles/style.css" />
   </head>
   <body class="body">
+    <div class="top-section">
```

Comment:
are you really need 2 comtainers here "top-section" and "img-top"
if you want you can create section-container and reuse it for all sections. you can set for example max-width/width/margin/ for all setions